hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -604,4 +596,34 @@ public void setConf(Configuration conf) {
   public Configuration getConf() {
     return conf;
   }
+
+  @VisibleForTesting
+  HiveClientPool loadHiveClientPool(Configuration configuration) {
```

Comment:
Could we just call it `clientPool`? I would prefer not to "leak" information about how it is handled internally

Reply:
Fixed it.